contracts/VaultV3.vy

```diff
@@ -130,7 +130,7 @@ symbol: public(String[32])
 # `nonces` track `permit` approvals with signature.
 nonces: public(HashMap[address, uint256])
 DOMAIN_SEPARATOR: public(bytes32)
-DOMAIN_TYPE_HASH: constant(bytes32) = keccak256('EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)')
+DOMAIN_TYPE_HASH: constant(bytes32) = keccak256('EIP712Domain(string name,string version,uint256 chain_id,address verifying_contract)')
```

Comment:
is this correct? wondering if this changes the DOMAIN_TYPE_HASH and if it is important or not